src/tests/parser.test.ts

```
@@ -3,7 +3,7 @@
 import { TestCaseLoader } from '../parser/loader.js';
 import { MessageRole } from '../core/types/message.types.js';
 import { ParseError } from '../parser/errors.js';
-import { writeFile, mkdir, readdir, unlink, rmdir } from 'fs/promises';
+import { writeFile, mkdir, readdir, unlink, rmdir, rm } from 'fs/promises';
```

ESLint

[error] 'unlink' is defined but never used.

(@typescript-eslint/no-unused-vars)